hs_core/templates/pages/my-resources.html

```diff
@@ -353,92 +353,95 @@ <h4 class="modal-title" id="myModalLabel">Create New Label</h4>
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
+                            aria-hidden="true">&times;</span></button>
+                    <h4 class="modal-title" id="myModalLabel">Manage Labels</h4>
+                </div>
+                <div class="modal-body">
+                    <table id="table-user-labels" class="table">
+                        <thead>
+                        <tr>
+                            <th>Label</th>
+                            <th></th>
                         </tr>
-                    {% endif %}
-                    </tbody>
-                </table>
-            </div>
-            <div class="modal-footer">
-                <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                        </thead>
+                        <tbody>
+                        {% if user.ulabels.saved_labels %}
+                            {% for label in user.ulabels.saved_labels %}
+                                <tr>
+                                    <td class="user-label" data-label="{{ label }}">{{ label }}</td>
+                                    <td>
+                                        <form class="hidden-form" data-id="form-delete-label-{{ label }}"
+                                              action="/hsapi/_internal/label-resource-action/"
+                                              method="post">
+                                            {% csrf_token %}
+                                            <input type="hidden" name="label" value="{{ label }}">
+                                            <input type="hidden" name="label_type" value="SAVEDLABEL">
+                                            <input type="hidden" name="action" value="DELETE">
+                                        </form>
+                                        <span data-label="{{ label }}" data-form-type="delete-label"
+                                              class="btn-label-remove glyphicon glyphicon-remove"
+                                              data-form-id="form-delete-label-{{ label }}"></span>
+                                    </td>
+                                </tr>
+                            {% endfor %}
+                        {% else %}
+                            <tr class="no-items-found">
+                                <td>No labels found.</td>
+                            </tr>
+                        {% endif %}
+                        </tbody>
+                    </table>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
```

Comment:
id field. Chrome will generate warning for missing ID.

Reply:
@engineSound this button does not need an id attribute at the moment. If it is needed for automated testing I think the id should be added once the test is written.

Reply:
Automation testing shouldn't touch source code.